.idea/misc.xml

```diff
@@ -10,4 +10,4 @@
   <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="mcreator_sdk" project-jdk-type="JavaSDK">
```

Comment:
Please revert changes in this file so it is not part of the PR.

Reply:
Sorry, I had problems with my branches and I had to re-add the SDK, but it had the wrong name. It should be ok now.